Scripts/Player.cs

```diff
@@ -161,6 +162,12 @@ public void Move(Vector2 weightedDirection, float delta)
         }
     }
 
+    public void WeaponFired(Projectile projectile) 
+    {
+        GD.Print(projectile.Velocity.Length());

```

Comment:
Harro

Reply:
Hmm i thought i fixed that. I think haven't pushed yet